package.json

```diff
@@ -102,7 +101,6 @@
     "test": "cross-env PARSE_BUILD=node jest",
     "test:mongodb": "npm run test:mongodb:runnerstart && npm run integration",
     "test:mongodb:runnerstart": "mongodb-runner start -- --port 27017",
-    "test:types": "dtslint --expectOnly types",
```

Comment:
I think I figured out how to enable full typescript support with shipping types to npm. I was waiting for #2425 to be merged so we don’t have conflicts